src/components/ButtonPanel.js

```diff
@@ -1,7 +1,49 @@
-export default function ButtonPanel() {
-  return (
-    <h1>
-      Button Panel
-    </h1>
-  );
-}
+import React from 'react';
+import Button from './buttons/button';
+
+// /* eslint-disable */
```

Comment:
- [ ] <b>REQUIRED:</b> Actually, this doesn't make any difference, `eslint-disable` keywords are designed to be comments. it doesn't matter what kind of comment is used as long as it is there. So kindly please remove it.

Reply:
It was not working actually